src/neptune/internal/operation_processors/offline_operation_processor.py

```diff
@@ -44,7 +44,8 @@ def __init__(self, container_id: UniqueId, container_type: ContainerType, lock:
 
     @staticmethod
     def _init_data_path(container_id: UniqueId, container_type: ContainerType) -> Path:
-        return Path(f"{NEPTUNE_DATA_DIRECTORY}/{OFFLINE_DIRECTORY}/{container_type.create_dir_name(container_id)}")
+        container_dir = get_container_dir(OFFLINE_DIRECTORY, container_id, container_type)
```

Comment:
This could be simplified after changing the return type of `get_container_id` to `Path`